File: Array/21_30/22Solution.java
class Solution {
    public List<List<Integer>> generate(int numRows) {
        List<List<Integer>> result=new ArrayList<>();
        List<Integer> list=new ArrayList<>();
        list.add(1);
        result.add(list);
        for(int i=1;i<numRows;i++){
            list=new ArrayList<>();
            int start=0;
            for(int j=0;j<=i;j++){
                if(j==0||j==i){
                    list.add(1);
                    continue;
                }list.add(result.get(i-1).get(start)+result.get(i-1).get(start+1));
                start++;
            }
            result.add(list);
        }
        return result;
    }
}

File: Array/21_30/22Solution.py
def generate(numRows):
        m=[]
        l=[[1],[1,1]]
        if numRows==1:
            return([[1]])
        elif numRows==2:
            return([[1],[1,1]])
        else:
            for i in range(2,numRows):
                m=[1]
                for j in range(1,i):
                    m.append(l[i-1][j-1]+l[i-1][j])
                m.append(1)
                l.append(m)
            return(l)
numRows=int(input())
print(generate(numRows))
